packages/opentelemetry-metrics/src/export/controllers/PushController.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConsoleLogger, ExportResult, unrefTimer } from '@opentelemetry/core';
+import * as types from '@opentelemetry/api';
+import { Resource } from '@opentelemetry/resources';
+import { Meter } from '../../';
+import { DEFAULT_CONFIG, MeterConfig } from '../../types';
+import { PushControllerConfig } from './types';
+import { NoopExporter } from '../NoopExporter';
+import { UngroupedBatcher } from '../Batcher';
+import { Controller } from '../Controller';
+
+const DEFAULT_EXPORT_INTERVAL = 60_000;
```

Comment:
think this should be `60000` ?

Reply:
Yeah, quite the same. This syntax is a [stage 3 proposal](https://github.com/tc39/proposal-numeric-separator) that already working in transpilers like TypeScript.

Reply:
This is not new code, previously at https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-metrics/src/export/Controller.ts#L21 moved here.

Reply:
Didn't knew about this syntax, thanks for explaining it :)